packages/tabs/src/TabTitle.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useRef, useEffect, useCallback, SetStateAction } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+const listTitle = css`
+  display: inline-block;
+  background-color: ${uiColors.white};
+  color: ${uiColors.gray.base};
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 1;
+  // padding-bottom dervied from border height + 5px
+  padding: 3px 20px 8px;
+  cursor: pointer;
+  transition: 150ms color ease-in-out;
+  border: none;
+  background: none;
+  text-decoration: none;
+
+  &:focus {
+    color: ${uiColors.blue.base};
+    outline: none;
+  }
+
+  &:hover:not(:focus) {
+    color: ${uiColors.gray.dark3};
+  }
+`;
+
+interface SharedTabTitleProps {
```

Comment:
I'm wondering if this should extend from `TabProps` as that's the public interface and this has to in many ways mirror what public props people are using.

Reply:
👍